extensions/wikia/Search/classes/Hooks.php

```diff
@@ -85,7 +93,19 @@ public static function onArticleSaveComplete(
 	 * @return true
 	 */
 	public static function onArticleUndelete( $title, $create ) {
-		return ( new Indexer )->reindexBatch( [ $title->getArticleID() ] );
+		if ( $this->canIndex($title) ) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Remove this use of "$this". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS2014)